src/main/java/de/srendi/advancedperipherals/common/addons/computercraft/peripheral/RSBridgePeripheral.java

```diff
@@ -0,0 +1,926 @@
+            if (inventory == null) {
+                return MethodResult.of(0, StatusConstants.INVENTORY_NOT_FOUND.name());
+            }
+        }
+
+        return importToRS(arguments, inventory);
+    }
+
+    @Override
+    @LuaFunction(mainThread = true)
+    public final MethodResult exportItem(IComputerAccess computer, IArguments arguments) throws LuaException {
+        if (!isAvailable())
+            return notConnected();
+
+        IItemHandler inventory = InventoryUtil.getHandlerFromDirection(arguments.getString(1), owner);
+
+        if (inventory == null) {
+            inventory = InventoryUtil.getHandlerFromName(computer, arguments.getString(1));
+            if (inventory == null) {
+                return MethodResult.of(0, StatusConstants.INVENTORY_NOT_FOUND.name());
+            }
+        }
+
+        return exportToChest(arguments, inventory);
+    }
+
+    @Override
+    @LuaFunction(mainThread = true)
+    public final MethodResult importFluid(IComputerAccess computer, IArguments arguments) throws LuaException {
+        if (!isAvailable())
+            return notConnected();
+
+        IFluidHandler handler = FluidUtil.getHandlerFromDirection(arguments.getString(1), owner);
+        if (handler == null) {
+            handler = FluidUtil.getHandlerFromName(computer, arguments.getString(1));
+            if (handler == null) {
+                return MethodResult.of(0, StatusConstants.INVENTORY_NOT_FOUND.name());
+            }
+        }
+
+        return importToRS(arguments, handler);
+    }
+
+    @Override
+    @LuaFunction(mainThread = true)
+    public final MethodResult exportFluid(IComputerAccess computer, IArguments arguments) throws LuaException {
+        if (!isAvailable())
+            return notConnected();
+
+        IFluidHandler handler = FluidUtil.getHandlerFromDirection(arguments.getString(1), owner);
+        if (handler == null) {
+            handler = FluidUtil.getHandlerFromName(computer, arguments.getString(1));
+            if (handler == null) {
+                return MethodResult.of(0, StatusConstants.INVENTORY_NOT_FOUND.name());
+            }
+        }
+
+        return exportToTank(arguments, handler);
+    }
+
+    @Override
+    @LuaFunction(mainThread = true)
+    public final MethodResult importChemical(IComputerAccess computer, IArguments arguments) throws LuaException {
+        if (!isAvailable())
+            return notConnected();
+
+        if (!APAddons.refinedStorageMekanismLoaded)
+            return MethodResult.of(Collections.emptyList());
+
+        IChemicalHandler handler = ChemicalUtil.getHandlerFromDirection(arguments.getString(1), owner);
+        if (handler == null) {
+            handler = ChemicalUtil.getHandlerFromName(computer, arguments.getString(1));
+            if (handler == null) {
+                return MethodResult.of(0, StatusConstants.INVENTORY_NOT_FOUND.name());
```

Comment:
`notConnected()` is returning `nil, err`, `!refinedStorageMekanismLoaded` is returning an empty list, and `handler == null` is returning `0, err`.

I think we should make a standard that always return `nil, err` when an error happens, unless the action can be partially done.

This affects all import/export methods

Reply:
I agree
I changed it, so every method has a specific default value which will be documented 
The commit contains a description